WapTenantPublicAPI.psd1

```diff
@@ -8,174 +8,181 @@
-    'Get-WAPVMNetworkSubnet',
-    'Get-WAPVMNetworkSubnetIPPool',
-    'Get-WAPVMRole',
-    'Get-WAPVMRoleOSDisk',
-    'Get-WAPVMRoleVM',
-    'Get-WAPVMRoleVMSize',
-    'Grant-WAPVMNetworkAccess',
-    'New-WAPCloudService',
-    'New-WAPSQLDatabase',
-    'New-WAPVMNetwork',
-    'New-WAPVMNetworkSubnet',
-    'New-WAPVMNetworkSubnetIPPool',
-    'New-WAPVMRoleDeployment',
-    'New-WAPVMRoleParameterObject',
-    'Remove-WAPCloudService',
-    'Remove-WAPSQLDatabase',
-    'Remove-WAPVMNetwork',
-    'Remove-WAPVMNetworkSubnet',
-    'Remove-WAPVMNetworkSubnetIPPool',
-    'Reset-WAPSQLDatabaseAdmin',
-    'Resize-WAPSQLDatabase',
-    'Select-WAPSQLOffer',
-    'Select-WAPSubscription',
-    'Set-WAPVMNetworkSubnetIPPool',
-    'Start-WAPVM',
-    'Stop-WAPVM',
-    'Test-WAPSQLDatabaseNameAvailable',
-    'Get-WAPWebSpace',
-    'Test-WAPWebSiteNameAvailable',
-    'Get-WAPWebSite',
-    'Select-WAPWebSpace',
-    'New-WAPWebSite',
-    'Remove-WAPWebSite',
-    'Get-WAPWebSiteConfiguration',
-    'Get-WAPWebSitePublishingXML',
-    'Restart-WAPWebSite',
-    'New-WAPWebSiteGitRepository',
-    'Get-WAPWebSiteGitRepository',
-    'Remove-WAPWebSiteGitRepository',
-    'Get-WAPWebSitePublishingInfo',
-    'Get-WAPAdminSubscription',
-    'Get-WAPAdminCloud',
-    'Get-WAPVMTemplate',
-    'Remove-WAPVM',
-    'New-WAPVM'
-)
-
-# Cmdlets to export from this module
-# CmdletsToExport = '*'
-
-# Variables to export from this module
-VariablesToExport = @(
-    'Token',
-    'Headers'
-)
-
-# Aliases to export from this module
-# AliasesToExport = '*'
-
-# DSC resources to export from this module
-# DscResourcesToExport = @()
-
-# List of all modules packaged with this module
-# ModuleList = @()
-
-# List of all files packaged with this module
-# FileList = @()
-
-# Private data to pass to the module specified in RootModule/ModuleToProcess. This may also contain a PSData hashtable with additional module metadata used by PowerShell.
-PrivateData = @{
-
-    PSData = @{
-
-        # Tags applied to this module. These help with module discovery in online galleries.
-        Tags = @('WAPack','VMRole','AzurePack','PSModule','DBaaS')
-
-        # A URL to the license for this module.
-        LicenseUri = 'https://github.com/bgelens/WAPTenantPublicAPI/blob/master/LICENSE'
-
-        # A URL to the main website for this project.
-        ProjectUri = 'https://github.com/bgelens/WAPTenantPublicAPI'
-
-        # A URL to an icon representing this module.
-        # IconUri = ''
-
-        # ReleaseNotes of this module
-        # ReleaseNotes = ''
-
-    } # End of PSData hashtable
-
-} # End of PrivateData hashtable
-
-# HelpInfo URI of this module
-HelpInfoURI = 'https://github.com/bgelens/WAPTenantPublicAPI'
-
-# Default prefix for commands exported from this module. Override the default prefix using Import-Module -Prefix.
-# DefaultCommandPrefix = ''
+    # Script module or binary module file associated with this manifest.

```

Comment:
I think you auto formatted the manifest using vs code ps ext default settings  (alignment around the = sign) . I'm not in favor of this. Could you revert?

Reply:
done